app/src/main/java/com/rootstrap/android/ui/activity/main/targetpoint/MapFragment.kt

```diff
@@ -90,6 +100,48 @@ class MapFragment : PermissionFragment(), OnMapReadyCallback {
         setMapCamera(location)
     }
 
+    private fun addCircleMarkerForTarget(target: TargetModel) {
+        val position = LatLng(target.lat, target.lng)
+        target.topic?.run {
+            mMap.addMarker(
+                MarkerOptions()
+                    .position(position)
+                    .icon(bitmapDescriptorwithOvalBackground(requireContext(), getIconForTarget()))
+            )
+        }
+    }
+
+    private fun bitmapDescriptorwithOvalBackground(context: Context, @DrawableRes vectorDrawableResourceId: Int): BitmapDescriptor? {
```

Comment:
why do you need the context, if you are already in a fragment, you should use the fragment context.

Reply:
yes, true.